articles/active-directory-b2c/openid-connect-technical-profile.md

```diff
@@ -120,7 +121,8 @@ The **CryptographicKeys** element contains the following attribute:
 
 | Attribute | Required | Description |
 | --------- | -------- | ----------- |
-| client_secret | Yes | The client secret of the identity provider application. The cryptographic key is required only if the **response_types** metadata is set to `code`. In this case, Azure AD B2C makes another call to exchange the authorization code for an access token. If the metadata is set to `id_token` you can omit the cryptographic key.  |
+| client_secret | Yes | The client secret of the identity provider application. This cryptographic key is required only if the **response_types** metadata is set to `code` and **token_endpoint_auth_method** is set to `client_secret_post` or `client_secret_basic`. In this case, Azure AD B2C makes another call to exchange the authorization code for an access token. If the metadata is set to `id_token` you can omit the cryptographic key.  |
```

Comment:
Could we add `private_key_jwt` as one of the options as well?

Reply:
@HarmanDhunna , can you clarify what you mean? I thought **client_secret** was only used for `client_secret_post` and `client_secret_basic` and that **assertion_signing_key** was used for `private_key_jwt`?

Reply:
Ah yes. you are correct. Confused myself.